ttrpc-codegen/src/lib.rs

```diff
@@ -132,11 +136,18 @@ impl Codegen {
         let includes: Vec<&Path> = self.includes.iter().map(|p| p.as_path()).collect();
         let inputs: Vec<&Path> = self.inputs.iter().map(|p| p.as_path()).collect();
         let p = parse_and_typecheck(&includes, &inputs)?;
+        // Add default path from env OUT_DIR, if no OUT_DIR env ,that's will be current path
+        let path_dir = std::env::var("OUT_DIR").map_or_else(
+            |_| std::env::current_dir().unwrap_or_default(),
+            PathBuf::from,
+        );
+        // If out_dir is none ,dst_path will be setting in path_dir
+        let dst_path = self.out_dir.clone().unwrap_or(path_dir);
```

Comment:
How about using unwrap_or_else to improve performance. 
```
        // If out_dir is none ,dst_path will be setting in path_dir
        let dst_path = self.out_dir.clone().unwrap_or_else(|| {
            // Add default path from env OUT_DIR, if no OUT_DIR env ,that's will be current path
            let path_dir = std::env::var("OUT_DIR").map_or_else(
                |_| std::env::current_dir().unwrap_or_default(),
                PathBuf::from,
            );

            path_dir
        });
```

Reply:
OK change it